### PR TITLE
METRICS bnf decription was broken

### PR DESCRIPTION
**Pull request by geraldguillaume**
Problem: f21956b5bf1b9d325af0b530b144564c6a84e030 commit broke METRICS description
Signed-off-by: Gerald Guillaume <geraldguillaume@eaton.com>

### CODE REVIEW

**Comment by EldoreiJK** [comment]
Just checking, isn't .bnf autogenerated?

File: src/fty_proto.bnf [hunk 1]
```diff
@@ -7,18 +7,17 @@ The following ABNF grammar defines the Core FTY protocols:
...
-
-Value defines duration of validity (i.e. how long is metric valid,
-when it expires) as well as the latest time we should receive metric
-updates.
     type            = string                ; Metric name, e.g.: "temperature", "humidity", "power.load", ...
     name            = string                ; Name of asset where metric is produced.
     value           = string                ; Metric value, e.g.: "25.323" or "900".
     unit            = string                ; Metric unit, e.g.: "C" or "F" for temperature.
 
+Value defines duration of validity (i.e. how long is metric valid,
```
Comment (jimklimov) [inline src/fty_proto.bnf:17]: Looking at the referenced commit, the comment about "Value defined ..." should be up there near the TTL field; I am not sure how it should be formatted (e.g. indented or prefixed with semicolons) to make a proper BNF markup. If indentation matters, then maybe the end of line introduced before "unixtime" was the problem.

Is there some parser to check validity of a BNF text?

**Review by jimklimov — CHANGES_REQUESTED** [review]

File: src/fty_proto.bnf (hunk 1 above)
Comment (jimklimov) [reply src/fty_proto.bnf:17]: I found a http://www.icosaedro.it/bnf_chk/bnf_chk-on-line.html which complains on every other line we have (the non-empty ones ;P) but generally it seems that windy comment paragraphs' lines not starting with a comment character (semicolon or hash as that tool insists) are a major problem.

Comment (jimklimov) [reply src/fty_proto.bnf:17]: Note that Pieter mentioned that BNF in his experience was a buzzword for something that some strange people want to put into their documentation books to pass some customer acceptance requirements but never use in practice, e.g. for machine parsing, although they could... so it was likely never a priority to make the generator a very correct one.

**Pull request closed without merging**